client/extensions/woocommerce/app/index.js

```diff
@@ -25,22 +25,26 @@ import { isLoaded as arePluginsLoaded } from 'state/plugins/installed/selectors'
 import { isStoreSetupComplete } from 'woocommerce/state/sites/setup-choices/selectors';
 import Main from 'components/main';
 import Placeholder from './dashboard/placeholder';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
 import QueryJetpackPlugins from 'components/data/query-jetpack-plugins';
 import RequiredPluginsInstallView from 'woocommerce/app/dashboard/required-plugins-install-view';
 import WooCommerceColophon from 'woocommerce/components/woocommerce-colophon';
-import PageViewTracker from 'lib/analytics/page-view-tracker';
 
 class App extends Component {
 	static propTypes = {
-		siteId: PropTypes.number,
+		allRequiredPluginsActive: PropTypes.bool,
+		analyticsPath: PropTypes.string,
+		analyticsTitle: PropTypes.string,
 		canUserManageOptions: PropTypes.bool.isRequired,
 		children: PropTypes.element.isRequired,
 		documentTitle: PropTypes.string,
+		fetchSetupChoices: PropTypes.func.isRequired,
 		hasPendingAutomatedTransfer: PropTypes.bool.isRequired,
 		isAtomicSite: PropTypes.bool.isRequired,
 		isDashboard: PropTypes.bool.isRequired,
-		analyticsPath: PropTypes.string,
-		analyticsTitle: PropTypes.string,
+		pluginsLoaded: PropTypes.bool.isRequired,
+		siteId: PropTypes.number,
+		translate: PropTypes.func.isRequired,
 	};
```

Comment:
If I understand this correctly, compared to the other changes to this file, it should only reorder stuff, and yet here I see 3 lines being removed and 7 being added. 😄 
Just to confirm: are those being added because they were missing before, or is it a rebase leftover?

Reply:
They were missing – `allRequiredPluginsActive`, `fetchSetupChoices`, `pluginsLoaded`, and `translate` were not in this list before, the other 3 are reordered.